### DISCUSSION TITLE
AssetMapper debbing javascript files won't florlesly work with PHPStorm

### DISCUSSION THREAD

**Discussion by leon-manq** [symfony/symfony#58597] [Q&A]
I've been working with AssetMapper for a small project, and so far, it's worked pretty well. I really like the JetBrains IDEs, but I've had some issues debugging the JavaScript files, the breakpoints don't work.

Is this because AssetMapper doesn't generate source maps? In the development environment, a hash is added to the JavaScript files to prevent caching issues, if I'm correct. As a result, PHPStorm seems unable to create a proper mapping to the files. Could this be why debugging JavaScript files with AssetMapper doesn't work properly?
As a workaround, I’m currently using debugger statements.

**Comment by sbarex** [comment]
I think it is due to the fact that the js file is handled by asset mapper changing the name and path and consequently the breakpoint inserted in the original file is not activated because the file name does not match.

It would still be useful to find a workaround for the problem!
